packages/commands/serve/src/index.ts

```diff
@@ -1,103 +1,18 @@
-    name: 'serve',
-  };
-};
-
-export type PlaygroundConfig = GraphQLPlaygroundConfig & {
-  tabs: { [name: string]: RenderPageOptions['tabs'][0] };
-}
-
-export type MockConfig = { [typeName: string]: string };
-export type ServeConfig = { mocks: MockConfig; playground: PlaygroundConfig; } ;
-
-export async function loadMocks(mockConfig?: MockConfig) {
-  if(!mockConfig) {
-    return {};
-  }
-  const mocks: IMocks = {};
-  const mocksLoad$: Promise<void>[] = [];
-  for (const typeName in mockConfig) {
-    const [ moduleName, importName ] = mockConfig[typeName];
-    mocksLoad$.push(
-      import(moduleName).then(
-        mod => mocks[typeName] = mod[importName]
-      )
-    )
-  }
-  await Promise.all(mocksLoad$);
-  return mocks;
-}
-
-export const plugin: CliPlugin = {
-  init({ program, loadProjectConfig, reportError }) {
-    program
-      .command('serve [port]')
-      .action(async (port: string | number = '4000') => {
-        try {
-
-          const config = await loadProjectConfig({
-            extensions: [ServeExtension]
-          });
-
-          const serveConfig: ServeConfig = await config.extension('serve');
-
-          const [ schema, mocks ] = await Promise.all([
-            config.getSchema(),
-            serveConfig.mocks && loadMocks(serveConfig.mocks)
-          ]);
-
-          let playground: GraphQLPlaygroundConfig = true;
-          if (serveConfig.playground && serveConfig.playground.tabs) {
-            const normalizedPlaygroundConfig: any = {};
-            if (typeof serveConfig.playground === 'object') {
-              Object.assign(normalizedPlaygroundConfig, serveConfig.playground);
-            }
-            const normalizedTabs = [];
-            for (const tabName in serveConfig.playground.tabs) {
-              normalizedTabs.push({
-                name: tabName,
-                ...serveConfig.playground.tabs[tabName]
-              });
-            }
-            normalizedPlaygroundConfig.tabs = normalizedTabs;
-            playground = normalizedPlaygroundConfig;
-          }
-
-          const apolloServer = new ApolloServer({
-            schema,
-            mocks,
-            playground,
-          });
-
-          const serverInfo = await apolloServer.listen(port)
-          
-          const url = `http://localhost:${port}/graphql`;
-          console.info(`Serving the GraphQL API and Playground on ${url}`);
-          console.info(JSON.stringify(serverInfo, null, 2));
-          await open(url);
-
-        } catch (e) {
-          reportError(e);
-        }
-      });
-  }
-};
+import { defineCommand } from '@test-graphql-cli/common';
+import { loaders } from '@test-graphql-cli/loaders';
+import serve from '@graphql-inspector/serve-command';
+import { LoadersRegistry } from '@graphql-inspector/loaders';
+
+export default defineCommand(() => {
+  const loadersRegistry = new LoadersRegistry();
+
+  loaders.forEach((loader) => loadersRegistry.register(loader));
+
+  return serve({
```

Comment:
This command is replacement of old graphql-cli's playground command and serving it with mocks should be optional in that case. Also in the previous implementation we allowed people to add their own mock implementations.